hieradata/clients/private.vpn.jenkins.io.yaml

```
@@ -2,17 +2,17 @@
 profile::openvpn::networks:
   eth0:
     # Defined in https://github.com/jenkins-infra/azure-net/blob/main/vpn.tf
-    name: "'private-vnet-vpn' subnet from 'private-vnet' virtual network"
+    name: "'dmz' subnet from 'private-vnet' virtual network"
     route-metric: 100
     network_cidr: 10.248.0.0/28
   eth1:
     # Defined in https://github.com/jenkins-infra/azure-net/blob/main/vnet.tf
-    name: "'private-vnet' virtual network"
+    name: "'private-vnet-data-tier' subnet from 'private-vnet' virtual network"
     route-metric: 200
-    network_cidr: 10.248.0.0/14
+    network_cidr: 10.248.1.0/24
```

```suggestion
    network_cidr: 10.248.1.0/24
    # azure-net:vnets.tf/private-vnet
    peered_network: 10.248.0.0/14
```
Should we peer the private virtual network here?